src/Components/AuthSignup/index.js

```diff
@@ -3,11 +3,14 @@ import "./styles.css";
 import { Link } from "react-router-dom";
 import { useAuth } from "../../Contexts/AuthContext";
 import { useState } from "react";
-import axios from "axios";
-
+import { addUser } from "../../Call-Apis/add-user";
 import { useNavigate } from "react-router-dom";
 
 function AuthSignup() {
+  const EMAIL_REGEX = new RegExp(
+    "^(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|\"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*\")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\\[(?:(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9]))\\.){3}(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9])|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\\])$"
+  );
+
```

Comment:
you can save it somewhere  else and then use it instead of repeating it 2 times in login and signup

Reply:
It's just one line and I've only used it two places so I thought we don't need to create new folder just for that , that's why i did not do it